easybuild/easyconfigs/p/psmpi/psmpi-5.1.5-1-iccifort-2015.1.133.eb

```diff
@@ -0,0 +1,27 @@
+name = 'psmpi'
+version = '5.1.5-1'
+
+homepage = 'https://github.com/ParaStation/psmpi2'
+description = """ParaStation MPI is an open source high-performance MPI 3.0 implementation,
+based on MPICH v3. It provides extra low level communication libraries and integration with 
+various batch systems for tighter process control.
+"""
+
+toolchain = {'version': '2015.1.133', 'name': 'iccifort'}
+
+toolchainopts = {'defaultopt': True}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/ParaStation/psmpi2/archive/']
+
+dependencies = [
+    ('pscom', '5.0.48-1', '', True),
+]
+
+hiddendependencies = [
+    ('pscom', '5.0.48-1', '', True),
+]
+
+configopts = ' --with-confset=intel PSCOM_LDFLAGS=-L$EBROOTPSCOM/lib PSCOM_CPPFLAGS=-I$EBROOTPSCOM/include --with-mpichconf="--enable-static" '
```

Comment:
since the value for `--with-confset` is determine by the compiler being used, this should really go in the easyblock...